recipes/RECOLA/train.py

```diff
@@ -0,0 +1,186 @@
+import sys
+import speechbrain as sb
+from hyperpyyaml import load_hyperpyyaml
+from expBrain import expBrain
+import pandas as pd
+import numpy as np
+from preprocess import preprocess_RECOLA
+import torch
+
+
+def main():
```

Comment:
Ok, I see you have the main function here in a `train.py` - why is there a need for a seperated `run_all.py` ?